src/components/Products.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from "react";
+import styled, { ThemeProvider } from "styled-components";
+import products from "../data/featured-products.json";
+import productCategories from "../data/product-categories.json";
+import FeaturedProducts from "./FeaturedProducts";
+import "./Products.css";
+function Products() {
+  const [productItems, setProductItems] = useState(products.results);
+  const [currentCategory, setCurrentCategory] = useState([]);
+
+  // Define our `fg` and `bg` on the theme
+  const theme = {
+    fg: "#00004f",
+    bg: "white",
+  };
+
+  // This theme swaps `fg` and `bg`
+  const invertTheme = ({ fg, bg }) => ({
+    fg: bg,
+    bg: fg,
+  });
+  const DivWrapper = styled.div``;
+
+  const DivSideBar = styled.div`
+    float: left;
+    width: 20%;
+  `;
+
+  const DivMain = styled.div`
+    width: 80%;
+  `;
+
+  const List = styled.ul`
+    list-style-type: none;
+    margin: 10;
+    padding: 10;
+    width: 200px;
+    background-color: ${(props) => props.theme.bg};
+  `;
+
+  const StyledItem = styled.a`
+    color: ${(props) => props.theme.fg};
+    background: ${(props) => props.theme.bg};
+
+    display: block;
+    padding: 8px 16px;
+    text-decoration: none;
+
+    &:hover {
+      background-color: ${(props) => props.theme.fg};
+      color: ${(props) => props.theme.bg};
+    }
+  `;
+
+  const handleChangeCategory = (event) => {
+    let category = event.target.name.toLowerCase();
```

Comment:
if category is not going to be modified i suggest you to make the variable a constant